foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/mutating/webhook/ConfigMutatingWebhook.java

```
@@ -33,31 +48,124 @@
+
+
+            if (target.length() > 1) {
+                JsonNode patch = JsonDiff.asJson(source, mapper.readTree(target.toString()));
+                response = new AdmissionResponseBuilder()
+                        .withAllowed(true)
+                        .withUid(request.getUid())
+                        .withPatch(Base64.getEncoder().encodeToString(patch.toString().getBytes()))
+                        .withPatchType("JSONPatch")
+                        .build();
+                logger.info("The kubernetes resource request has been updated.");
+            } else {
+                response = new AdmissionResponseBuilder()
+                        .withAllowed(true)
+                        .withUid(request.getUid())
+                        .build();
+                logger.info("No updates have made to the kubernetes resource request.");
+            }
+
+        } catch (Exception e) {
+            logger.error("Error injecting the config store value", e);
+            // create failed response
+            response = new AdmissionResponseBuilder()
+                    .withAllowed(false)
+                    .withUid(request.getUid())
+                    .withStatus(new StatusBuilder().withStatus("Failed").build())
+                    .build();
+        }
+
+        return new AdmissionReviewBuilder()
+                .withResponse(response)
+                .withApiVersion(admissionReviewRequest.getApiVersion())
+                .build();
+    }
+
+    public String getProperty(String groupName,String propertyName) {
+        PropertyKey propertyKey = new PropertyKey(groupName, propertyName);
+        Property property;
+        try {
+            property = this.configLoader.read(propertyKey);
+            return property.getValue();
+        } catch (Exception e) {
+            logger.error(String.format("Error reading property - groupName: %s, propertyName: %s", groupName, propertyName), e);
```

S: We should just re-throw an exception here and allow it to bubble up to the top where we return the fail response to k8s.